File: app/src/main/java/com/example/starwars/model/Genre.java
package com.example.starwars.model;

import com.google.gson.annotations.SerializedName;

public class Genre {
    @SerializedName("name")
    private String name;

    public String getName() {
        return name;
    }
    public static String convertToString(Genre[] genres) {
        StringBuilder builder = new StringBuilder();
        for (Genre genre: genres) {
            builder.append(genre.getName()).append(", ");
        }
        builder.deleteCharAt(builder.length() - 1);
        builder.deleteCharAt(builder.length() - 1);
        return builder.toString();
    }
}

File: app/src/main/java/com/example/starwars/model/Genre.kt
package com.example.starwars.model

import com.google.gson.annotations.SerializedName
import com.example.starwars.model.Genre
import java.lang.StringBuilder

class Genre {
    @SerializedName("name")
    val name: String? = null

    companion object {
        fun convertToString(genres: Array<Genre>?): String {
            if (genres == null) return ""
            val builder = StringBuilder()
            for (genre in genres) {
                builder.append(genre.name).append(", ")
            }
            builder.deleteCharAt(builder.length - 1)
            builder.deleteCharAt(builder.length - 1)
            return builder.toString()
        }
    }
}